src/models/OrdersModel.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Commerce Insights Orders Model
+ *
+ * @link      https://fostercommerce.com
+ * @copyright Copyright (c) 2021 Foster Commerce
+ */
+
+declare(strict_types = 1);
+
+namespace fostercommerce\commerceinsights\models;
+
+use fostercommerce\commerceinsights\helpers\Helpers;
+
+use craft\base\Model;
+
+class OrdersModel extends Model
```

Comment:
Currently, `OrderModel` extends `Model`, but doesn't actually do anything. It defines static factory methods to transform data into an associative array. My suggestion is to use the model to _hold_ the data, so that it the data can be accessed like `$orderModel->reference` or 

```php
foreach ($orderModel->lineItems as $lineItem) {
  var_dump($lineItem->sku);
}
```

Use the model to define the data.